### ISSUE TITLE
Record/playback mode.

### ISSUE THREAD

**Issue by rec** [ManiacalLabs/SimPixel#20]
We need a way to create stand-alone demos that run this program on the web without having an instance of Bibliopixel.

Ideally, something we can just drop into a web page somehow...!

**Comment by adammhaile** [comment]
Will discuss with @mwcz 
His project Kimotion already has something like this. We'd just need a way to generate the data and save it to a file from BP, which should be simple. But yes, when you go to simpixel.io it should start up a recorded demo to make it more interesting. Maybe this can come in when we handle Issue #2

**Comment by mwcz** [comment]
With Kimotion, we have a command that starts up the server, but instead of opening a websocket server, it pipes the outgoing bytes into a raw binary file.  Then [Kimotion's recording parser](https://github.com/mwcz/Kimotion/blob/master/src/kinect-recording.js) loads up that recording and steps through it.

It was exceptionally easy because every Kimotion frame is the same length.  With SimPixel, the client will need some way to know how long each frame is, since there's no temporal spacing between the frames when they come from a binary file.  Perhaps a small header field in the configuration frame with the number of LEDs?  I think that would be sufficient.

**Comment by adammhaile** [comment]
Yeah, as with Kimotion, the frames are all the same length. The main things we need are length and framerate, as what comes from BiblioPixel is variable. I can easily write a data recorder driver that can store the data with a header. How hard do you think it would be to get the client side to read that file at a specific rate?

**Comment by adammhaile** [comment]
@mwcz - I think you did some of this already? What do you need from BiblioPixel to make this happen. I'm ok with this being a v1.1 feature... really more for first load anyways.
